dashboard/apps/prototype/management/commands/cache.py

```diff
@@ -25,6 +26,7 @@ def generate(self):
         """
         generate cache
         """
+        cache_tools.METHOD_CACHE_WRITE_ONLY = True
```

Comment:
a 3rd solution is to split the celery specific settings from `base.py` and put it into its own module, where we can just set `METHOD_CACHE_WRITE_ONLY = True`. celery will only generate the cache and not consume it.

another thing to be aware is that the impact of `METHOD_CACHE_WRITE_ONLY` only affects `method_cache` decorator, and doesn't affect anything else, e.g. `page_cache`.